src/components/History/Transaction.js

```diff
@@ -7,13 +7,17 @@
  */
 
 import React, { Component } from 'react';
-import { Text, View } from 'react-native';
+import { LayoutAnimation, Text, View, Platform, UIManager } from 'react-native';
 import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
 import { History as t } from '../../utils/i18n';
 import colors from '../../styles/colors';
 import { floatToEuro } from '../../utils/amount';
 import { beautifyDate, beautifyDateTime } from '../../utils/date';
 
+if (Platform.OS === 'android' && UIManager.setLayoutAnimationEnabledExperimental) {
+	UIManager.setLayoutAnimationEnabledExperimental(true);
```

Comment:
You cannot define a component setting in a sub file. Do it directly in the main app file pls